### DISCUSSION TITLE
Github secret and Token secret should be public??

### DISCUSSION THREAD

**Discussion by 7hacker** [elegantframework/elegant-cli#567] [General]
The docs state that the following environment variables should be set

NEXT_PUBLIC_CMS_GITHUB_ID="your-github-client-id"
NEXT_PUBLIC_CMS_GITHUB_SECRET="your-github-client-secret"
NEXT_PUBLIC_CMS_TOKEN_SECRET="random-32-digit-string"

Vercel usually warns you from setting secret keys with NEXT_PUBLIC like so: "This key, which is prefixed with NEXT_PUBLIC_ and includes the term SECRET, might expose sensitive information to the browser. Verify it is safe to share publicly."

so how does this work? is this safe or best practice?

**Comment by Bowens20832** [comment]
@7hacker Thank you for your help and patience, this should be resolved now in #569.

What do you think?
